docs/src/main/paradox/openshift.md

```diff
@@ -31,17 +32,24 @@ guide uses `centralpark.lightbend.com` as an example, you will have to use your
 
 ### Running
 
-Login to OpenShift from your terminal and create the OpenShift project:
+First, let's prepare a few environment variables to make things easier:
 
-```bash
+```
 ## obtain the token from https://<your-openshift-server>/console/command-line
-export OPENSHIFT_SERVER=centralpark.lightbend.com
 export TOKEN=<my-token>
+export OPENSHIFT_SERVER=centralpark.lightbend.com
```

Comment:
Maybe a note saying that centralpark.lightbend.com is our testing environment and that this should be replaced by their own.

Reply:
Or maybe simply this:

```suggestion
export OPENSHIFT_SERVER=<your-openshift-server>
```

Reply:
> Maybe a note saying that centralpark.lightbend.com is our testing environment and that this should be replaced by their own.

Hmmm, I had it. Did I overwrite it on a rebase/CONFLICT? Damn it! Thanks for noticing!

Reply:
Ah, it's still there: https://github.com/playframework/play-scala-grpc-example/pull/46/files#diff-ee560919a91b5db403f596de2c67860dR30